website/content/docs/upgrade/upgrade-specific.mdx

```diff
@@ -21,6 +21,13 @@ Nomad 1.9.0 no longer supports the HCLv1 format for job specifications. Using
 the `-hcl1` option for the `job run`, `job plan`, and `job validate` commands
 will no longer work.
 
+#### Default `infra_image` changed
```

Comment:
Let's make it really clear this only impacts Docker so people who don't use Docker can breath a sigh of release and not read any further.

```suggestion
#### Default Docker `infra_image` changed
```